docs/lua-modding/lua-modding-tools.mdx

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 2
+---
+
+# Lua Modding Tools
+If you already know the tools, you can skip ahead, but don't blame us if we reference something and you don't know what we're talking about.
+
+
+## **[UE4SS](https://github.com/UE4SS-RE/RE-UE4SS)**
+---
+This is your bread and butter. UE4SS is a scripting service for unreal engine which allows us to even load the mods at all. It also comes with a console that is extremely helpful for developing and testing mods.
+### **Installation**
+---
+If you've used any mods you probably have this installed already, but it's not exactly covered there, so we'll cover it here.
```

Comment:
maybe add a note that if they're using a mod manager they probably already have ue4ss